### PR TITLE
Update banners that upsell Jetpack products

### PR DESCRIPTION
**Pull request by monsieur-z**
### Changes proposed in this Pull Request

This PR updates the banners that upsell Jetpack products and plans, according to the new offer.

Fixes: 1169247016322522-as-1187641228151000

### Implementation notes

- Updated the icons to match Calypso
- Refactored `PlanIcon` to use these icons
- Created a map to link features and product/plan upsells, so that subsequent updates are easy to make

_TODO: update jetpack.com redirects in WPcom_

### Testing instructions

- Select a self-hosted Jetpack site with a Free plan
- Run the plugin locally
- Check that each banner in the table below has been updated (copy, colour, and icon)
- In _Settings > Security > Backups and security scanning_, check that you see a different banner with a Security Daily plan

### Screenshots

Screen Shot 2020-09-22 at 4 41 39 PM

### Does this pull request change what data or activity we track or use?
No

### Proposed changelog entry for your changes:
Update banners that upsell Jetpack plans and products to reflect the new offer

### CODE REVIEW

File: _inc/client/components/banner/style.scss [hunk 1]
```diff
@@ -60,7 +70,7 @@
 		}
 	}
 
-	@include breakpoint( ">480px" ) {
+	@include breakpoint( '>480px' ) {
```
Comment (monsieur-z) [inline _inc/client/components/banner/style.scss:73]: Automatic formatting.

File: _inc/client/components/settings-card/index.jsx [hunk 2]
```diff
@@ -137,18 +135,22 @@ export const SettingsCard = props => {
 			case FEATURE_SECURITY_SCANNING_JETPACK:
 				if (
 					backupsEnabled ||
-					'is-business-plan' === planClass ||
-					'is-complete-plan' === planClass ||
+					[ 'is-business-plan', 'is-realtime-security-plan', 'is-complete-plan' ].includes(
```
Comment (monsieur-z) [inline _inc/client/components/settings-card/index.jsx:138]: These plans include Backup Real-time already.

File: _inc/client/components/settings-card/index.jsx [hunk 3]
```diff
@@ -137,18 +135,22 @@ export const SettingsCard = props => {
...
+						planClass
+					) ||
 					props.multisite
 				) {
 					return '';
 				}
 
-				if ( 'is-premium-plan' === planClass ) {
+				if ( [ 'is-premium-plan', 'is-daily-security-plan' ].includes( planClass ) ) {
```
Comment (monsieur-z) [inline _inc/client/components/settings-card/index.jsx:146]: We want to promote Backup Real-time if customer owns these plans.

File: _inc/client/lib/plans/constants.js [hunk 4]
```diff
@@ -57,6 +57,20 @@ export const JETPACK_MONTHLY_PLANS = [
...
 ];
+export const JETPACK_OFFER_RESET_PLANS = [
+	PLAN_JETPACK_SECURITY_DAILY,
+	PLAN_JETPACK_SECURITY_DAILY_MONTHLY,
+	PLAN_JETPACK_SECURITY_REALTIME,
+	PLAN_JETPACK_SECURITY_REALTIME_MONTHLY,
+	PLAN_JETPACK_COMPLETE,
+	PLAN_JETPACK_COMPLETE_MONTHLY,
+];
+export const JETPACK_BUNDLES = [
```
Comment (monsieur-z) [inline _inc/client/lib/plans/constants.js:68]: Bundles are shown in a different colour (purple).

File: _inc/client/scss/variables/_colors.scss [hunk 5]
```diff
@@ -37,71 +37,75 @@ $gray-text-min:          darken( $gray, 18% ); //#537994
 //
 // See wordpress.com/design-handbook/colors/ for more info.
 
+// Product types
```
Comment (monsieur-z) [inline _inc/client/scss/variables/_colors.scss:40]: Only addition to this file. The rest is automatic formatting.

File: _inc/client/at-a-glance/scan.jsx [hunk 6]
```diff
@@ -180,13 +184,15 @@ class DashScan extends Component {
 		return (
 			<JetpackBanner
 				callToAction={ __( 'Upgrade', 'jetpack' ) }
-				title={ __( 'Find threats early so we can help fix them fast.', 'jetpack' ) }
+				title={ __(
+					'Automated scanning and one-click fixes keep your site ahead of security threats.',
```
Comment (rcanepa) [inline _inc/client/at-a-glance/scan.jsx:188]: Just out of curiosity, where is this copy change coming from? Does it come from the Offer Reset copy document?

**Review by rcanepa — APPROVED** [review]
LGTM! I like very much that we went from having everything hard-coded to something configurable.

File: _inc/client/at-a-glance/scan.jsx (hunk 6 above)
Comment (monsieur-z) [reply _inc/client/at-a-glance/scan.jsx:188]: I've just added the link in the ticket.

Comment (rcanepa) [reply _inc/client/at-a-glance/scan.jsx:188]: Thank you!

File: _inc/client/lib/plans/constants.js [hunk 7]
```diff
@@ -57,6 +57,20 @@ export const JETPACK_MONTHLY_PLANS = [
 	PLAN_JETPACK_SECURITY_REALTIME_MONTHLY,
 	PLAN_JETPACK_COMPLETE_MONTHLY,
 ];
+export const JETPACK_OFFER_RESET_PLANS = [
```
Comment (jeherve) [inline _inc/client/lib/plans/constants.js:60]: I wonder if this (and the other mentions of "offer reset") should be named differently. It makes sense now, but on the long run I don't know if "offer reset" will mean much, as we keep iterating on plans.

**Review by jeherve — APPROVED** [review]
This tests well for me. I only have one remark, not a blocker. It may be worth iterating on this in a follow-up PR.

File: _inc/client/lib/plans/constants.js (hunk 7 above)
Comment (monsieur-z) [reply _inc/client/lib/plans/constants.js:60]: You're right @jeherve. I think a better way of doing here could be to invert the logic and refer to legacy plans instead. I'll take a not to handle that in another PR.

**Pull request merged**